src/System.CommandLine/Builder/ArgumentExtensions.cs

```diff
@@ -61,6 +61,17 @@ public static Argument<DirectoryInfo> ExistingOnly(this Argument<DirectoryInfo>
             return argument;
         }
 
+        public static Argument<FileSystemInfo> ExistingOnly(this Argument<FileSystemInfo> argument)
```

Comment:
For consistency with the existing methods, could you also add an equivalent for `Argument<FileSystemInfo[]>`?

Reply:
Done!